frontend/stories/schema/schema.graphql

```diff
@@ -0,0 +1,2054 @@
+# All input for the `authenticate` mutation.
```

Comment:
I don't think we should have these files checked in.

In .travis.yml I have it generating the schema from the endpoint before running the tests, it calls get-schema in the frontend package.json.